wiki/Glossary/BeatmapSetID/en.md

```diff
@@ -0,0 +1,18 @@
+---
+tags:
+  - links
+---
+
+# Beatmapset ID
+
+*See also: [BeatmapID](/wiki/Glossary/BeatmapID)*
+
+A *beatmapsetID* is an ID shared between all difficulties of a [beatmapset](/wiki/Beatmaps/Beatmapsets). This number is a unique, whole number which can never be changed. This allows for permanent navigation to and from a beatmapset. The beatmapsetID can be found in the url of a beatmapset on osu! web as the first set of numbers, or in the `.osu` difficulty file.
```

Comment:
```suggestion
A **beatmapsetID** is an ID shared between all difficulties of a [beatmapset](/wiki/Beatmaps/Beatmapsets). This number is a unique, whole number which can *never be changed*. This allows for permanent navigation to and from a beatmapset. The beatmapsetID can be found in the url of a beatmapset on the osu! website as the first set of numbers, or in the `.osu` difficulty file.
```
Same deal as above; I'd love to have an image highlighting those first set of numbers.